code/index.html

```diff
@@ -1,22 +1,93 @@
 <!DOCTYPE html>
 <html>
+
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>Travel insights</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content= "width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/6f68b8218c.js" crossorigin="anonymous"></script>
   </head>
+
   <body>
     <header>
-      <!-- Some kind of logo and navbar -->
+      <img src="suitcase-logo1.jpeg" alt="Airplane logo">
+      <div class="site-name">TRAVEL <br> INSIGHTS
```

Comment:
If you want to replace the br tag (I think I read about it here https://www.notion.so/Week-2-common-pitfalls-bbe695289d154ff98d6a247cef873f07) I think you can use two h1 or h2 with the same class. Maybe something like: 

```suggestion
<div>
  <h1 class="site-name">TRAVEL</h1>
  <h1 class="site-name">INSIGHTS</h1>
</div>
```

and it will automatically be a line break I think (but ofc I might be wrong!)